src/SearchInput/__snapshots__/SearchInput.test.js.snap

```diff
@@ -16,8 +16,12 @@ exports[`<SearchInput /> create SearchInput 1`] = `
         className="fd-popover__control"
       >
         <div
+          aria-haspopup={true}
           className="fd-input-group fd-input-group--after"
           onClick={[Function]}
+          onKeyPress={[Function]}
```

Comment:
Similar behavior as `ComboboxInput` where `space` key presses are consumed to open the popup.